routing/global_shortest/GlobalRouter.cpp

```diff
@@ -1400,18 +1400,18 @@ std::string GlobalRouter::GetRoutingInfoFile()
      if (nav_line_file == "")
           return nav_line_file;
      else
-          return _building->GetProjectRootDir()+nav_line_file;
+          return _building->GetProjectRootDir() / nav_line_file;
 }
 
 
-bool GlobalRouter::LoadRoutingInfos(const std::string &filename)
+bool GlobalRouter::LoadRoutingInfos(const fs::path &filename)
 {
-     if(filename=="") return true;
+     if(filename.empty()) return true;
 
      Log->Write("INFO:\tLoading extra routing information for the global/quickest path router");
-     Log->Write("INFO:\t  from the file "+filename);
+     Log->Write("INFO:\t  from the file "+filename.string());
 
-     TiXmlDocument docRouting(filename);
+     TiXmlDocument docRouting(filename.c_str());
```

Comment:
@Ozaq Many thanks for these work !

- I got an error here , and after changing to filename.string(), I compile jpscore on windows successfully. I have tested demos/scenario_3_corner, it works. 

- And there is another small problem, which is the location of geometry file in trajectory file is wrong. I have attached a screenshot here, I think it's easy to fix.

![image](https://user-images.githubusercontent.com/32732507/63225078-f1755b00-c1cc-11e9-974d-686c9e9de1b5.png)

Reply:
Thanks for testing, ill have a look

Reply:
@xuqiancheng It should(TM) work now

Reply:
@Ozaq Yes, it works now !